docs/docsite/rst/porting_guide_2.3.rst

```diff
@@ -0,0 +1,193 @@
+.. _porting_2.1_guide:
+
+*************************
+Ansible 2.3 Porting Guide
+*************************
+
+This section discusses the changes to how you use Ansible between Ansible 2.2 and Ansible 2.3.
+
+It is intended to assist in updating your playbooks, plugins, etc so they will work with this version of Ansible.
+
+
+We suggest you read this page along with https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#23-ramble-on---2017-04-12 before upgrading Ansible to understand what, if any updates you will need to make.
```

Comment:
"to understand what (if any) updates you will need to make."  Although removing the "(if any)" altogether might be better.